### DISCUSSION TITLE
Cleaning up snapshots

### DISCUSSION THREAD

**Discussion by AdamLeyshon** [mitsuhiko/insta#670] [Q&A] [answered]
Hey team, how does one purge old snapshots where tests have been removed or renamed?

I checked the help/issues pages but I didn't find any sort of `cargo insta clean` command

Thanks, WP/

**Comment by max-sixty** [comment, marked as answer]
`cargo insta test --unreferenced=delete` 😀

**Reply by AdamLeyshon** [reply]
🤯 wow after re-reading https://insta.rs/docs/cli/ I can't believe I missed that

Thanks for responding so quickly and apologies for not properly RTFM.

**Reply by max-sixty** [reply]
No prob!
